dex/asset.go

```diff
@@ -59,19 +59,21 @@ func init() {
 	}
 }
 
-// LockTimeTaker returns the taker locktime value that should be used by both
-// client and server for the specified network. Mainnet uses a constant value
-// while test networks support setting a custom value during build.
+// LockTimeTaker returns the taker locktime value (i.e. how long funds are locked
+// for in swap contract before they can be refunded) that client and server will
```

Comment:
There's no need to add the definition of locktime to the docs for these functions. Locktime is a fundamental concept for atomic swaps, and is well-defined via other documentation.

Reply:
Does it hurt to be slightly repetitive with such documentation ? The locktime concept is obvious to someone who maintained this code for a while, for a newcomer looking at this code for the very first time it might not be 100% clear whether it is The locktime (or he might not even be familiar with the concept yet and is just browsing through the code, that's how I started).

Reply:
it's perfectly clear

Reply:
Okay, reverted my updates to those comments.